viewing_party/party.py

```diff
@@ -1,23 +1,168 @@
+# -----------------------------------------
 # ------------- WAVE 1 --------------------
+# -----------------------------------------
+from csv import list_dialects
+
+
+TITLE_KEY = "title"
+GENRE_KEY = "genre"
+RATING_KEY = "rating"
+
 
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {
+        TITLE_KEY: title,
+        GENRE_KEY: genre,
+        RATING_KEY: rating}
+
+    if not new_movie[TITLE_KEY]:
+        return None
+    if not new_movie[GENRE_KEY]:
+        return None
+    if not new_movie[RATING_KEY]:
+        return None
+    return new_movie
+
+
+def add_to_watched(user_data, movie):
+    user_data = {"watched": [movie]}
+    return user_data
+
+
+def add_to_watchlist(user_data, movie):
+    user_data = {"watchlist": [movie]}
+    return user_data
+
+
+def watch_movie(user_data, title):
+    if user_data["watchlist"]:
+        for movie in user_data['watchlist']:
+            if movie['title'] == title:
+                user_data["watched"].append(movie)
+                user_data['watchlist'].remove(movie)

```

Comment:
This one is a little confusing to me. I am not sure what you mean here and would love to go over it with you so I understand. I am looking at it and I think this was the one where i needed to append to the watched and remove it from the watchlist since the title is the title of the movie the user watched so it needs to modify the list of 'movies to watch' to 'movies that have been watched'. So I think I may be misunderstanding what you are saying.